### PR TITLE
Add inject req and pressure spec and reconcile logic 

### PR DESCRIPTION
**Pull request by moomman**
1. add spec to specify inject pressure and job
2. add reconcile logic to control

### CODE REVIEW

File: pitr/cli/.golangci.yml [hunk 1]
```diff
@@ -39,10 +40,30 @@ linters:
     - asciicheck
     - prealloc
     - stylecheck
+=======
```
Comment (mlycore) [inline pitr/cli/.golangci.yml:43]: Fix these conflicts please

**Review by mlycore — CHANGES_REQUESTED** [review]

File: pitr/cli/.golangci.yml (hunk 1 above)
Comment (moomman) [reply pitr/cli/.golangci.yml]: ok

File: shardingsphere-operator/pkg/controllers/shardingsphere_chaos_controller.go [hunk 2]
```diff
@@ -19,31 +19,301 @@ package controllers
...
+	batchV1 "k8s.io/api/batch/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	ShardingSphereChaosControllerName = "shardingSphereChaos-controller"
```
Comment (mlycore) [inline shardingsphere-operator/pkg/controllers/shardingsphere_chaos_controller.go:42]: Rename it to `shardingsphere-chaos-controller` keep it simple to read

File: shardingsphere-operator/api/v1alpha1/shardingsphere_chaos_types.go [hunk 3]
```diff
@@ -34,14 +36,201 @@ type ShardingSphereChaosList struct {
...
 	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              ShardingSphereChaosSpec   `json:"spec,omitempty"`
+	Status            ShardingSphereChaosStatus `json:"status,omitempty"`
+}
 
-	Spec ShardingSphereChaosSpec `json:"spec,omitempty"`
+// ShardingSphereChaosSpec defines the desired state of ShardingSphereChaos
+type ShardingSphereChaosSpec struct {
+	InjectJob  JobSpec   `json:"injectJob,omitempty"`
+	ChaosKind  ChaosKind `json:"chaosKind,omitempty"`
```
Comment (mlycore) [inline shardingsphere-operator/api/v1alpha1/shardingsphere_chaos_types.go:46]: Consider remove this `ChaosKind` and we have already make it explicit in `EmbedChaos`

**Review by mlycore — DISMISSED** [review]

File: shardingsphere-operator/pkg/controllers/shardingsphere_chaos_controller.go (hunk 2 above)
Comment (moomman) [reply shardingsphere-operator/pkg/controllers/shardingsphere_chaos_controller.go:42]: ok

File: shardingsphere-operator/api/v1alpha1/shardingsphere_chaos_types.go (hunk 3 above)
Comment (moomman) [reply shardingsphere-operator/api/v1alpha1/shardingsphere_chaos_types.go]: solved

**Review by mlycore — APPROVED** [review]

**Pull request merged**